src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private static WriterServiceImpl writerService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        writerService = new WriterServiceImpl();
+    }
+
+    @Test
+    public void validParams_Ok() throws IOException {
```

Comment:
```suggestion
    public void writeToFile_validData_Ok() throws IOException {
```

and so on
There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.

Reply:
Thank you, Daria!